service/managedinstance/providers/aws/aws.go

```diff
@@ -305,6 +305,23 @@ type DeleteManagedInstanceInput struct {
 
 type DeleteManagedInstanceOutput struct{}
 
+type StatusManagedInstanceInput struct {
+	ManagedInstanceID *string `json:"managedInstanceId,omitempty"`
+}
+
+type StatusManagedInstanceOutput struct {
+	ID           *string    `json:"id,omitempty"`
+	ImageId      *string    `json:"imageId,omitempty"`
```

Comment:
```suggestion
	ImageID      *string    `json:"imageId,omitempty"`
```

[Ref](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).